vignettes/spatial.Rmd

```diff
@@ -0,0 +1,258 @@
+---
+title: "CAR Spatial Models with rstanarm"
+author: "Imad Ali, Jonah Gabry and Ben Goodrich"
+date: "`r Sys.Date()`"
+output: 
+  html_vignette: 
+    toc: yes
+params:
+  EVAL: !r identical(Sys.getenv("NOT_CRAN"), "true")
+---
+<!--
+%\VignetteEngine{knitr::rmarkdown}
+%\VignetteEncoding{UTF-8}
+%\VignetteIndexEntry{stan_spatial: Models for Spatial Data}
+-->
+```{r, child="children/SETTINGS-knitr.txt"}
+```
+```{r, child="children/SETTINGS-gg.txt"}
+```
+```{r, child="children/SETTINGS-rstan.txt"}
+```
+```{r, child="children/SETTINGS-loo.txt"}
+```
+
+## Introduction
+
+This vignette explains how to account for spatial variation with conditional autoregressive (CAR) models when modeling discrete or continuous outcomes using the `stan_besag`, `stan_bym`, and `stan_bym2` functions in the __rstanarm__ package. The `stan_besag` function allows you to model space as an intrinsic conditional autoregressive model (ICAR), the `stan_bym` function allows you to fit the Besag, York, Mollie (1991) CAR model, and the `stan_bym2` is a variant of the BYM model where the priors are more interpretable (see Riebler et al (2016) for details).
+
+```{r, child="children/four_steps.txt"}
+```
+
+Steps 3 and 4 are covered in more depth by the vignette entitled ["How to Use
+the __rstanarm__ Package"](rstanarm.html). This vignette focuses on Step 1 when
+the likelihood is the product of beta distributions.
+
+## Likelihood
+
+The likelihoods supported in each CAR modeling function include Gaussian, Binomial, Poisson, Negative Binomial, and Gamma. For details on the link functions available and the contribution to the posterior of each of these likelihoods consult the appropriate [vignette](http://mc-stan.org/rstanarm/articles/index.html). 
+
+The linear predictor takes the following form,
+$$
+\boldsymbol{\eta} = \alpha + \mathbf{X}\boldsymbol{\beta} + \boldsymbol{\psi}
+$$
+where $\alpha$ is the intercept, $\mathbf{X}$ is an $N$-by-$K$ matrix of predictors ($N$ being the number of observations and $K$ being the number of predictors), $\boldsymbol{\beta}$ is a $K$-dimensional vector of regression coefficients, and $\boldsymbol{\psi}$ is a $N$-dimensional vector representing the spatial effect. The construction of $\boldsymbol{\psi}$ depends on the model, which is discussed in the relevant sections below.
+
+Depending on the choice of likelihood there may or may not be an additional auxiliary parameter $\gamma$ in the model (e.g. in a Gaussian likelihood this would be the variation of the data). With all these components, for some probability density/mass function $f$, we can state the general form of the likelihood as,
```

Comment:
Is $\gamma$ actually $\sigma$ in the output?

Reply:
It's `aux` in the stan model but I think this maps to different things depending on the likelihood (e.g. it's sigma in the gaussian case).